src/content/glossary/index.md

```diff
@@ -92,6 +92,12 @@ Decentralized application. At a minimum, it is a [smart contract](#smart-contrac
 
 See [non-fungible token (NFT)](#nft)
 
+### defi {#defi}
+
+Short for "decentralized finance," a broad category of [Dapps](#dapp) with the intent of providing a wide range of financial services in a decentralized, peer-to-peer manner, without the need for a trusted third party, but instead utilizing [smart contracts](#smart-contract). How truly decentralized a given defi application may be involves many factors, and the truth often exists somewhere on a spectrum between completely centralized, and completely decentralized.
```

Comment:
Due to the long sentence structure and technical subject matter this is quite hard to read. Can we break this definition up into smaller sentences? We also probably don't need the sentence on "how decentralized a given defi application may be" as this is relevant for dapps in general.  Maybe we zero in on the fact that defi aims to offer the crypto equivalent of financial services without any of the gatekeeping, so anyone with an internet connection can benefit from financial services.

Reply:
How does this sound?
`Short for "decentralized finance," a broad category of [Dapps](#dapp) aiming to provide financial services backed by the blockchain, without any gatekeepers, so anyone with an internet connection can participate.`